instruction.md

```diff
@@ -0,0 +1,81 @@
+## Project setup and usage
```

Comment:
Please make this file more prettier. Here is how it looks like now:
https://github.com/exadel-inc/CompreFace/blob/EFRC-740-instruction-file-writing/instruction.md